_includes/footer.html

```diff
@@ -1,14 +1,12 @@
 <footer class="site-footer">
   <div class="wrapper">
-    Fully open source on <a href="https://github.com/ios-factor/ios-factor.com">GitHub</a>
-
-    <br />
-
-    Project started by <a href="https://twitter.com/KrauseFx">Felix Krause</a>
-
-    <br />
-
-    This project is in no way affiliated with Apple
+    {{ site.languageNames[site.lang] }}
+    {% for lang in site.languageNames %}
```

Comment:
Do you think we could make this link to the specific page (e.g. a specific factor) when switching the language?

Reply:
e.g. that it links to 
```
/ko/prefer-local-over-remote
```